### PR TITLE
41 initialization method define background and text colors not randomgly

### PR DESCRIPTION
**Pull request by sergiomrebelo**
I create a script named ColorGenerator.js for creating colour palettes based on a random base colour. Genetic operators are also modified. We tested the integration with generation.cy.js script

### CODE REVIEW

**Comment by JJ** [comment]
What user history would this be?

File: src/client/Params.js [hunk 1]
```diff
@@ -8,10 +8,32 @@ export class Params {
         'ta', 'te', 'th', 'tr', 'uk', 'ur', 'vi', 'cy'
     ];
 
+
+    // JOIN
     static availableTypefaces = [
         'Amstelvar', 'Anybody', 'Barlow', 'Cabin', 'Emberly', 'Epilogue', 'IBMPlexSans', 'Inconsolata'
     ]
 
+    static availableTypefacesInfo = {
```
Comment (JJ) [inline src/client/Params.js:17]: When you define this, `availableTypefaces` is simply `Object.keys(AvailableTypefacesInfo`

File: src/client/Params.js [hunk 2]
```diff
@@ -39,7 +61,7 @@ export class Params {
 
     static typography = {
         defaultColor: `#000000`,
-        range: 0.1,
+        range: 0.05,
```
Comment (JJ) [inline src/client/Params.js:64]: If you don't justify this somehow, you can keep changing it every single commit without converging. Same applies to mostly every other literal.

File: src/client/app.js [hunk 3]
```diff
@@ -45,7 +45,12 @@ window.windowResized = () => {
     if (window.app.screen < 2) return null;
 }
 
-window.keyPressed = () => {}
+window.keyPressed = () => {
+    if (window.app.screen < 2) return null;
+    if (key.toUpperCase() === 'S') {
+        // window.app.save();
```
Comment (JJ) [inline src/client/app.js:51]: You should eliminate the whole if instead of leaving this noop

Comment (JJ) [reply src/client/app.js:51]: And, anyway, shouldn't `key` be in the argument? Is it a global variable?

File: src/client/controllers/Population.js [hunk 4]
```diff
@@ -70,12 +70,12 @@ export class Population {
         // crossover
         for (let i = eliteSize; i < this.params["evo"]["popSize"]; i++) {
             if (Math.random() <= this.params["evo"]["crossoverProb"]) {
-                const parentA = this.tournament(2);
-                const parentB = this.tournament(2);
+
+                const parentA = this.tournament(TOURNAMENT_SIZE);
```
Comment (JJ) [inline src/client/controllers/Population.js:74]: This is good, but is it related to this PR?

**Review by JJ — COMMENTED** [review]
Don't quite get the point of the whole PR, I'm afraid. You might want to turn my comments into issues to address them.

File: src/client/controllers/Population.js (hunk 4 above)
Comment (sergiomrebelo) [reply src/client/controllers/Population.js:74]: No. It is not related. I reviewed some code with this branch open and do not divide from it.

File: src/client/app.js (hunk 3 above)
Comment (sergiomrebelo) [reply src/client/app.js:51]: Yes. My fault. I used this toe export the images to testing the colour schemes and forget to remove the if. [`key` ](https://p5js.org/reference/#/p5/key) is a global variable of `p5.js`, which I'm using for generate the canvas.

Comment (sergiomrebelo) [reply src/client/app.js:51]: I'm working in a commit to resolve that.

File: src/client/Params.js (hunk 1 above)
Comment (sergiomrebelo) [reply src/client/Params.js:17]: Yes, I know. This file is quite confusing. I am not well-estimating the necessary variables to configure the system. Currently, I just opened a new issue #45 to refactor it when the MVP is created.

File: src/client/Params.js (hunk 2 above)
Comment (sergiomrebelo) [reply src/client/Params.js:64]: I changed this to preliminary testing the results generation. I still need to test the parameters and define the experimental setup of the system. I just opened a new issue #47 to document that.

**Comment by sergiomrebelo** [comment]
> What user history would this be?

While this is not directly connected to any specific user story, it aligns with the overall objective of enhancing the system’s exploratory capabilities, which is a key aspect referred to in the user stories. Due to the exploratory characteristics of the system (stories https://github.com/sergiomrebelo/evo-poster/issues/1, https://github.com/sergiomrebelo/evo-poster/issues/23 and https://github.com/sergiomrebelo/evo-poster/issues/5). 

> Don't quite get the point of the whole PR, I'm afraid. You might want to turn my comments into issues to address them.
I was checking and fixing something that I found in this branch. It is a little confusing. My fault!

**Pull request merged**